docs/unleash.md

```
@@ -171,3 +178,10 @@ to this, however, so read the fine print.
 The alternative is to create feature flags one at a time via
 [`/api/admin/projects/:projectId/features`](https://docs.getunleash.io/reference/api/unleash/create-feature)
 which does accept a admin API token.
+
+## Production
+
+App SRE creates the feature flags for us based on files under
+`data/dependencies/unleash/feature_toggles/insights/rhsm/`.  Each file creates a
+single feature flag.  The particulars are discussed
+[here](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/platform-users/platform-services/unleash-feature-toggles.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)